.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: python
 python:
   - "3.3"
+  - "3.4"
+  - "3.5"
+cache: pip
 install:
-  - pip install -r requirements.txt
+  - pip install coveralls
   - python setup.py install
-script: nosetests -v askomics --with-doctest --with-coverage --cover-package=askomics
+script: python setup.py nosetests -v --with-doctest --with-coverage --cover-package=askomics
```

Comment:
Hum, shouldn't there be an && or something like that (or am I missing something obvious?)